ui/app/templates/vault/cluster/auth.hbs

```diff
@@ -49,52 +49,25 @@
     {{/if}}
   </Page.header>
   {{#unless this.mfaAuthData}}
-    {{#if this.managedNamespaceRoot}}
-      <Page.sub-header>
-        <Toolbar>
-          <div class="toolbar-namespace-picker" data-test-managed-namespace-toolbar>
-            <div class="field is-horizontal">
-              <div class="field-label">
-                <label class="is-label" for="namespace">Namespace</label>
-              </div>
-              <div class="field-label">
-                <span class="has-text-grey" data-test-managed-namespace-root>/{{this.managedNamespaceRoot}}</span>
-              </div>
-              <div class="field-body">
-                <div class="field">
-                  <div class="control">
-                    <input
-                      value={{this.managedNamespaceChild}}
-                      placeholder="/ (Default)"
-                      oninput={{perform this.updateManagedNamespace value="target.value"}}
-                      autocomplete="off"
-                      spellcheck="false"
-                      name="namespace"
-                      id="namespace"
-                      class="input"
-                      type="text"
-                    />
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-        </Toolbar>
-      </Page.sub-header>
-    {{else if (has-feature "Namespaces")}}
+    {{#if (has-feature "Namespaces")}}
       <Page.sub-header>
         <Toolbar class="toolbar-namespace-picker">
           <div class="field is-horizontal" data-test-namespace-toolbar>
             <div class="field-label is-normal">
               <label class="is-label" for="namespace">Namespace</label>
             </div>
+            {{#if this.managedNamespaceRoot}}
+              <div class="field-label">
+                <span class="has-text-grey" data-test-managed-namespace-root>/{{this.managedNamespaceRoot}}</span>
```

Comment:
Based on the above it looks like the prefix of `admin/` is fixed meaning we provide the trailing slash here? I wonder what the best UX would be - to add it here?
```suggestion
                <span class="has-text-grey" data-test-managed-namespace-root>/{{this.managedNamespaceRoot}}/</span>
```

Reply:
Well, it's sort of smoke and mirrors because my impression is that most people  are in the `admin` namespace directly. It doesn't really matter where we're putting the slashes because it's getting cleaned up when the namespace is set, but I can check with design

Reply:
I think I should have worded my question better. I'm wondering what the desired expectation for the input is - is the user expected to provide the slash: `'/my-child-ns`? (That's how I interpret the current display)

Reply:
Desired expectation is that we have consistent behavior whether or not the user include a leading or trailing slash. I'll add this to UI sync so we can talk about it with design and PM